recipes/icqsol/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: icqsol
+  version: "0.3.20"
+
+about:
+  home: 'https://github.com/pletzer/icqsol'
+  summary: 'A collection of utilities for constructing complex geometries from primitive shapes'
+  license: MIT
+
+source:
+  git_url: https://github.com/pletzer/icqsol.git
+  git_rev: 05b229c
+
+build:
+  number: 0
+
+test:
+  # Python imports
+  imports:
+    - csg
+    - icqsol
+    - numpy
+    - triangle
+    - vtk
+
+requirements:
+  build:
+    - cmake
+    - numpy
+    - python
+    - pycsg ==0.3.12
```

Comment:
The thing is if someone updates a new package of pycsg and then you install icqsol you will get this new dependency. If this is working, everything is fine. If you expect a future version breaking this recipe you should bin a version. Maybe <= works, ofter >= is used as well.